deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -78,8 +86,65 @@ data:
     </match>
     <filter prometheus.datapoint>
       @type carbon_v2
-    </filter>    
+    </filter>
     <match prometheus.datapoint>
+      @type rewrite_tag_filter
```

Comment:
Is that possible to move this filter before transform to `carbon_v2`?
rule may be something like `key=job`, `pattern=^(.*)$`.
The reason is we may move to Prometheus format later so we don't want it impacted

Reply:
That makes sense. Let me try this and get back to you.

Reply:
It seems to have worked! resolving